lib/src/components/NxDropdown/NxDropdown.scss

```diff
@@ -126,17 +126,12 @@
 
 .nx-dropdown-link {
   color: var(--nx-color-link);
-
-  &:hover {
-    color: var(--nx-color-link-hover);
```

Comment:
The text link has darker blue when hover, should we add it back?

Reply:
> The text link has darker blue when hover, should we add it back?

Yup! I don't know why i removed that 🤔. Thanks for catching that here's the commit: aa26b63eef59d56acde5231d24b62d69b9eea3e7